.github/workflows/ci.yml

```diff
@@ -94,7 +94,7 @@ jobs:
       - name: Install Doxygen
         run: |
           wget -qO- "http://doxygen.nl/files/doxygen-1.9.1.linux.bin.tar.gz" | sudo tar --strip-components=1 -xz -C /usr/local
-          sudo apt-get install -y libclang-9-dev
+          sudo apt-get install -y libclang-9-dev libclang-cpp9 graphviz
```

Comment:
You can just update this repository to use the shared action instead!

Reply:
Sure, I was thinking of fixing this for now as PRs are failing this check. I can follow up with a cleaned up ci.yml using the common github actions repository. Would that be okay?